### ISSUE TITLE
Re Importing Mesh in SP

### ISSUE THREAD

**Issue by MrPositivity** [DigiKrafting/blender_addon_substance_painter#11]
Hello, and thank you for this simple Plugin. 

I have tried other SP plugins, and this is one is the most simple one and does exactly what I need. 
I would wish to request one additional thing. 
Is it possible to Implement mesh reimporting to SP? 
Right now if I try to re-export from blender to SP, It will open new Substance Painter, can it be made to detect if SP is open and if it is you just Export mesh and update already opened  SP?

Positivity

**Comment by DigiKrafting** [comment]
Unfortunately the requested functionality which I would also like is not possible at this time as it would need to be implemented on the "Substance Painter" side of things. Reference: [https://docs.substance3d.com/spdoc/command-lines-98959411.html](https://docs.substance3d.com/spdoc/command-lines-98959411.html) I've posted the suggestion over at the Substance forums [Feedback](http://feedback.substance3d.com/forums/261284-substance-painter/suggestions/39844714-detect-already-open-project-and-update-mesh-instea) so add your vote.

**Comment by MrPositivity** [comment]
Yes, the more involved feature would be great, but maybe you could make it so if there is already .fbx exported you don't open substance but just update the .fbx? This way I can just Update my .fbx from painter .. Because now after opening the model in Substance, I need to Export manually if I wanna update mesh.

**Comment by DigiKrafting** [comment]
Hi, I've already added a parameter "open_sp" to the export functions, just want to give some thought and do some research on how best to detect/trigger the mesh export without adding extra buttons for it.
